src/Tasks/Build/Base.php

```diff
@@ -314,6 +314,51 @@ public function getDate()
         return date('Y-m-d');
     }
 
+    /**
+     * Copy the media files for an extension to the build folder
+     *
+     * @param   string  $name  The name of the extension to copy media files for
+     *
+     * @return  bool
+     *
+     * @since   1.0
+     */
+    public function buildMediaFiles($name)
+    {
+        $verbosity = $this->verbosityThreshold();
+        $this->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE);
+        $type   = substr($name, 0, 3);
+        $source = $this->getSourceFolder() . '/media/' . $name;
+        $target = $this->getBuildFolder() . '/media/' . $name;
+
+        if ($type == 'lib') {
+            $source = $this->getSourceFolder() . '/media/' . substr($name, 3);
+            $target = $this->getBuildFolder() . '/media/' . substr($name, 3);
+        }
```

Comment:
I'm not 100% sure if we should force it they way or allow both variants?

Reply:
I'd like to be oppionated and to force one way.